### DISCUSSION TITLE
Any presets other than medium quality resulted in stalled ray process

### DISCUSSION THREAD

**Discussion by Minhao-Zhang** [autogluon/autogluon#4172] [Q&A]
## Environment 
I am using the docker image nvcr.io/nvidia/pytorch:24.04-py3 and installed AutoGluon using pip. 

I have a RTX 2060 (notebook) GPU. 

## Issue 
Whenever I tried to train a model using TabularPredictor, fit will run smoothly with the preset 'medium_quality' with CPU usage shown below. The training finished in half an hour or so. 
![image](https://github.com/autogluon/autogluon/assets/41777700/e27d8233-e04f-4c10-aa1e-6ac236c2aeb5)

However, if I try to use any other preset, I will get the terminal output of 
```
Detecting stacked overfitting by sub-fitting AutoGluon on the input data. That is, copies of AutoGluon will be sub-fit on subset(s) of the data. Then, the holdout validation data is used to detect stacked overfitting.
Sub-fit(s) time limit is: 3600 seconds.
Starting holdout-based sub-fit for dynamic stacking. Context path is: .........
Running the sub-fit in a ray process to avoid memory leakage.
```
and nothing happens. The CPU usage is very low and I do not know why. 
![image](https://github.com/autogluon/autogluon/assets/41777700/360fc1b2-9e88-4d36-9d4b-e2e2b29a387f)

I have tried with a time_limit and without one, I have tried using GPU or not using GPU, I have even tried with a fresh docker container, but nothing seems to help. If you could give me a suggestion or point me to a documentation, that would be a great help.

**Comment by jshin1089** [comment]
Likewise, I installed AutoGluon using pip and started seeing "Running the sub-fit in a ray process to avoid memory leakage" when I would set the presets to anything other than "medium_quality". I even just left it on for a whole day, and no progress.

**Comment by tonyhoo** [comment]
Thanks for reporting this issue. Can you share the package versions by running following codes:
```
# Replace this code with the output of the following:
from autogluon.core.utils import show_versions
show_versions()
```

**Reply by jshin1089** [reply]
INSTALLED VERSIONS
------------------
date                   : 2024-05-13
time                   : 21:58:51.670291
python                 : 3.10.14.final.0
OS                     : Linux
OS-release             : 6.1.0-21-cloud-amd64
Version                : #1 SMP PREEMPT_DYNAMIC Debian 6.1.90-1 (2024-05-03)
machine                : x86_64
processor              : 
num_cores              : 16
cpu_ram_mb             : 104635.33203125
cuda version           : 12.525.125.06
num_gpus               : 1
gpu_ram_mb             : [14925]
avail_disk_size_mb     : 44006

accelerate             : 0.21.0
autogluon              : 1.1.0
autogluon.common       : 1.1.0
autogluon.core         : 1.1.0
autogluon.features     : 1.1.0
autogluon.multimodal   : 1.1.0
autogluon.tabular      : 1.1.0
autogluon.timeseries   : 1.1.0
boto3                  : 1.34.103
catboost               : 1.2.5
defusedxml             : 0.7.1
evaluate               : 0.4.2
fastai                 : 2.7.15
gluonts                : 0.14.3
hyperopt               : 0.2.7
imodels                : None
jinja2                 : 3.1.4
joblib                 : 1.4.2
jsonschema             : 4.21.1
lightgbm               : 4.3.0
lightning              : 2.1.4
matplotlib             : 3.8.4
mlforecast             : 0.10.0
networkx               : 3.3
nlpaug                 : 1.1.11
nltk                   : 3.8.1
nptyping               : 2.4.1
numpy                  : 1.26.4
nvidia-ml-py3          : 7.352.0
omegaconf              : 2.2.3
onnxruntime-gpu        : None
openmim                : 0.3.9
optimum                : 1.18.1
optimum-intel          : None
orjson                 : 3.10.3
pandas                 : 2.2.2
pdf2image              : 1.17.0
Pillow                 : 10.3.0
psutil                 : 5.9.8
pytesseract            : 0.3.10
pytorch-lightning      : 2.1.4
pytorch-metric-learning: 2.3.0
ray                    : 2.10.0
requests               : 2.28.2
scikit-image           : 0.20.0
scikit-learn           : 1.4.0
scikit-learn-intelex   : None
scipy                  : 1.12.0
seqeval                : 1.2.2
setuptools             : 60.2.0
skl2onnx               : None
statsforecast          : 1.4.0
tabpfn                 : None
tensorboard            : 2.16.2
text-unidecode         : 1.3
timm                   : 0.9.16
torch                  : 2.1.2
torchmetrics           : 1.2.1
torchvision            : 0.16.2
tqdm                   : 4.65.2
transformers           : 4.38.2
utilsforecast          : 0.0.10
vowpalwabbit           : None
xgboost                : 2.0.3

**Reply by Minhao-Zhang** [reply]
INSTALLED VERSIONS
------------------
date                   : 2024-05-15
time                   : 09:43:22.041954
python                 : 3.10.12.final.0
OS                     : Linux
OS-release             : 5.15.133.1-microsoft-standard-WSL2
Version                : #1 SMP Thu Oct 5 21:02:42 UTC 2023
machine                : x86_64
processor              : x86_64
num_cores              : 16
cpu_ram_mb             : 19686.3125
cuda version           : 12.550.73.01
num_gpus               : 1
gpu_ram_mb             : [4790]
avail_disk_size_mb     : 327273

accelerate             : 0.21.0
autogluon              : 1.1.0
autogluon.common       : 1.1.0
autogluon.core         : 1.1.0
autogluon.features     : 1.1.0
autogluon.multimodal   : 1.1.0
autogluon.tabular      : 1.1.0
autogluon.timeseries   : 1.1.0
boto3                  : 1.34.98
catboost               : 1.2.5
defusedxml             : 0.7.1
evaluate               : 0.4.2
fastai                 : 2.7.15
gluonts                : 0.14.3
hyperopt               : 0.2.7
imodels                : None
jinja2                 : 3.1.3
joblib                 : 1.3.2
jsonschema             : 4.21.1
lightgbm               : 4.3.0
lightning              : 2.1.4
matplotlib             : 3.8.4
mlforecast             : 0.10.0
networkx               : 3.3
nlpaug                 : 1.1.11
nltk                   : 3.8.1
nptyping               : 2.4.1
numpy                  : 1.26.4
nvidia-ml-py3          : 7.352.0
omegaconf              : 2.2.3
onnxruntime-gpu        : None
openmim                : 0.3.9
optimum                : 1.18.1
optimum-intel          : None
orjson                 : 3.10.3
pandas                 : 2.2.2
pdf2image              : 1.17.0
Pillow                 : 10.2.0
psutil                 : 5.9.4
pytesseract            : 0.3.10
pytorch-lightning      : 2.1.4
pytorch-metric-learning: 2.3.0
ray                    : 2.10.0
requests               : 2.28.2
scikit-image           : 0.20.0
scikit-learn           : 1.4.0
scikit-learn-intelex   : None
scipy                  : 1.12.0
seqeval                : 1.2.2
setuptools             : 60.2.0
skl2onnx               : None
statsforecast          : 1.4.0
tabpfn                 : None
tensorboard            : 2.9.0
text-unidecode         : 1.3
timm                   : 0.9.16
torch                  : 2.1.2
torchmetrics           : 1.2.1
torchvision            : 0.16.2
tqdm                   : 4.65.2
transformers           : 4.38.2
utilsforecast          : 0.0.10
vowpalwabbit           : None
xgboost                : 2.0.3

**Comment by e8742493** [comment]
I also have this issue
